influx-writer/influx-writer.py

```diff
@@ -117,16 +118,17 @@ def write(self, listen_dicts):
 
     def start(self):
         self.log.info("InfluxWriterSubscriber started")
+        print("InfluxWriterSubscriber started")
```

Comment:
We have a logger. Why can't we set up a `StreamHandler` to send the same data to the console too??
https://docs.python.org/2/howto/logging-cookbook.html